detectable/src/test/java/com/synopsys/integration/detectable/detectables/pnpm/functional/PnpmLockDetectableTest.java

```diff
@@ -37,7 +37,7 @@ protected void setup() throws IOException {
 
         addFile(
             Paths.get("pnpm-lock.yaml"),
-            "lockfileVersion: 1.0.0",
```

Comment:
Are versions 5 and 6 the only acceptable ones now?
If not, is it worthwhile to run this test for more than one version?

Reply:
v5 and v6 are the only lockfile versions we support right now. I left this file pretty much alone to handle v5 and created PnpmLockDetectableTestv6 to handle v6. I could maybe rename this file to make it clearer it is v5?

Reply:
Rename sounds reasonably useful and consistent with the newly-added v6 equivalent.